wrappers/golang/internal/generator/ntt/generate.go

```diff
@@ -13,12 +13,6 @@ var nttTemplates = map[string]string{
 	"header":       "ntt/templates/ntt.h.tmpl",
 }
 
-var ecnttTemplates = map[string]string{
```

Comment:
What happened here? Are they not needed anymore?

Reply:
it was a left over that wasn't removed. Ecntt generator was moved to its own folder